DEBUG 2016-03-31 07:02:47,830 utils (0.000) SET SQL_AUTO_IS_NULL = 0; args=None
DEBUG 2016-03-31 07:03:05,317 utils (0.000) SET SQL_AUTO_IS_NULL = 0; args=None
DEBUG 2016-03-31 07:03:05,344 utils (0.000) SET SQL_AUTO_IS_NULL = 0; args=None
DEBUG 2016-03-31 07:03:05,345 utils (0.001) SHOW FULL TABLES; args=None
DEBUG 2016-03-31 07:03:05,347 utils (0.000) SELECT `django_migrations`.`app`, `django_migrations`.`name` FROM `django_migrations`; args=()
DEBUG 2016-03-31 07:03:05,376 utils (0.001) 
            SELECT kc.`constraint_name`, kc.`column_name`,
                kc.`referenced_table_name`, kc.`referenced_column_name`
            FROM information_schema.key_column_usage AS kc
            WHERE
                kc.table_schema = 'benz' AND
                kc.table_name = 'portal_user'
        ; args=['benz', u'portal_user']
DEBUG 2016-03-31 07:03:05,377 utils (0.001) 
            SELECT c.constraint_name, c.constraint_type
            FROM information_schema.table_constraints AS c
            WHERE
                c.table_schema = 'benz' AND
                c.table_name = 'portal_user'
        ; args=['benz', u'portal_user']
DEBUG 2016-03-31 07:03:05,378 utils (0.000) SHOW INDEX FROM `portal_user`; args=None
DEBUG 2016-03-31 07:03:05,378 schema ALTER TABLE `portal_user` DROP FOREIGN KEY `portal_user_service_id_2ecc47d5_fk_portal_service_id`; (params [])
DEBUG 2016-03-31 07:03:05,786 utils (0.408) ALTER TABLE `portal_user` DROP FOREIGN KEY `portal_user_service_id_2ecc47d5_fk_portal_service_id`; args=[]
DEBUG 2016-03-31 07:03:05,787 schema ALTER TABLE `portal_user` DROP COLUMN `service_id` CASCADE; (params [])
DEBUG 2016-03-31 07:03:05,889 utils (0.102) ALTER TABLE `portal_user` DROP COLUMN `service_id` CASCADE; args=[]
DEBUG 2016-03-31 07:03:05,897 schema ALTER TABLE `portal_user` ADD COLUMN `service_name` varchar(256) NULL; (params [])
DEBUG 2016-03-31 07:03:05,957 utils (0.060) ALTER TABLE `portal_user` ADD COLUMN `service_name` varchar(256) NULL; args=[]
DEBUG 2016-03-31 07:03:05,958 schema ALTER TABLE `portal_user` ALTER COLUMN `service_name` DROP DEFAULT; (params [])
DEBUG 2016-03-31 07:03:05,966 utils (0.008) ALTER TABLE `portal_user` ALTER COLUMN `service_name` DROP DEFAULT; args=[]
DEBUG 2016-03-31 07:03:05,967 utils (0.001) SHOW FULL TABLES; args=None
DEBUG 2016-03-31 07:03:05,968 utils (0.000) INSERT INTO `django_migrations` (`app`, `name`, `applied`) VALUES ('portal', '0002_auto_20160331_0702', '2016-03-31 07:03:05.967964'); args=['portal', u'0002_auto_20160331_0702', u'2016-03-31 07:03:05.967964']
DEBUG 2016-03-31 07:03:05,985 utils (0.001) SHOW FULL TABLES; args=None
DEBUG 2016-03-31 07:03:05,986 utils (0.000) SELECT `django_migrations`.`app`, `django_migrations`.`name` FROM `django_migrations`; args=()
DEBUG 2016-03-31 07:03:05,988 utils (0.000) SELECT `django_content_type`.`id`, `django_content_type`.`app_label`, `django_content_type`.`model` FROM `django_content_type` WHERE (`django_content_type`.`model` = 'logentry' AND `django_content_type`.`app_label` = 'admin'); args=('logentry', 'admin')
DEBUG 2016-03-31 07:03:05,992 utils (0.000) SELECT `auth_permission`.`content_type_id`, `auth_permission`.`codename` FROM `auth_permission` INNER JOIN `django_content_type` ON (`auth_permission`.`content_type_id` = `django_content_type`.`id`) WHERE `auth_permission`.`content_type_id` IN (1) ORDER BY `django_content_type`.`app_label` ASC, `django_content_type`.`model` ASC, `auth_permission`.`codename` ASC; args=(1,)
DEBUG 2016-03-31 07:03:05,993 utils (0.000) SELECT `django_content_type`.`id`, `django_content_type`.`app_label`, `django_content_type`.`model` FROM `django_content_type` WHERE `django_content_type`.`app_label` = 'admin'; args=('admin',)
DEBUG 2016-03-31 07:03:05,995 utils (0.000) SELECT `django_content_type`.`id`, `django_content_type`.`app_label`, `django_content_type`.`model` FROM `django_content_type` WHERE (`django_content_type`.`model` = 'permission' AND `django_content_type`.`app_label` = 'auth'); args=('permission', 'auth')
DEBUG 2016-03-31 07:03:05,996 utils (0.000) SELECT `django_content_type`.`id`, `django_content_type`.`app_label`, `django_content_type`.`model` FROM `django_content_type` WHERE (`django_content_type`.`model` = 'group' AND `django_content_type`.`app_label` = 'auth'); args=('group', 'auth')
DEBUG 2016-03-31 07:03:05,998 utils (0.000) SELECT `django_content_type`.`id`, `django_content_type`.`app_label`, `django_content_type`.`model` FROM `django_content_type` WHERE (`django_content_type`.`model` = 'user' AND `django_content_type`.`app_label` = 'auth'); args=('user', 'auth')
DEBUG 2016-03-31 07:03:06,001 utils (0.001) SELECT `auth_permission`.`content_type_id`, `auth_permission`.`codename` FROM `auth_permission` INNER JOIN `django_content_type` ON (`auth_permission`.`content_type_id` = `django_content_type`.`id`) WHERE `auth_permission`.`content_type_id` IN (2, 3, 4) ORDER BY `django_content_type`.`app_label` ASC, `django_content_type`.`model` ASC, `auth_permission`.`codename` ASC; args=(2, 3, 4)
DEBUG 2016-03-31 07:03:06,002 utils (0.000) SELECT `django_content_type`.`id`, `django_content_type`.`app_label`, `django_content_type`.`model` FROM `django_content_type` WHERE `django_content_type`.`app_label` = 'auth'; args=('auth',)
DEBUG 2016-03-31 07:03:06,003 utils (0.000) SELECT `django_content_type`.`id`, `django_content_type`.`app_label`, `django_content_type`.`model` FROM `django_content_type` WHERE (`django_content_type`.`model` = 'contenttype' AND `django_content_type`.`app_label` = 'contenttypes'); args=('contenttype', 'contenttypes')
DEBUG 2016-03-31 07:03:06,006 utils (0.000) SELECT `auth_permission`.`content_type_id`, `auth_permission`.`codename` FROM `auth_permission` INNER JOIN `django_content_type` ON (`auth_permission`.`content_type_id` = `django_content_type`.`id`) WHERE `auth_permission`.`content_type_id` IN (5) ORDER BY `django_content_type`.`app_label` ASC, `django_content_type`.`model` ASC, `auth_permission`.`codename` ASC; args=(5,)
DEBUG 2016-03-31 07:03:06,007 utils (0.000) SELECT `django_content_type`.`id`, `django_content_type`.`app_label`, `django_content_type`.`model` FROM `django_content_type` WHERE `django_content_type`.`app_label` = 'contenttypes'; args=('contenttypes',)
DEBUG 2016-03-31 07:03:06,008 utils (0.000) SELECT `django_content_type`.`id`, `django_content_type`.`app_label`, `django_content_type`.`model` FROM `django_content_type` WHERE (`django_content_type`.`model` = 'session' AND `django_content_type`.`app_label` = 'sessions'); args=('session', 'sessions')
DEBUG 2016-03-31 07:03:06,011 utils (0.000) SELECT `auth_permission`.`content_type_id`, `auth_permission`.`codename` FROM `auth_permission` INNER JOIN `django_content_type` ON (`auth_permission`.`content_type_id` = `django_content_type`.`id`) WHERE `auth_permission`.`content_type_id` IN (6) ORDER BY `django_content_type`.`app_label` ASC, `django_content_type`.`model` ASC, `auth_permission`.`codename` ASC; args=(6,)
DEBUG 2016-03-31 07:03:06,012 utils (0.000) SELECT `django_content_type`.`id`, `django_content_type`.`app_label`, `django_content_type`.`model` FROM `django_content_type` WHERE `django_content_type`.`app_label` = 'sessions'; args=('sessions',)
DEBUG 2016-03-31 07:03:06,014 utils (0.000) SELECT `django_content_type`.`id`, `django_content_type`.`app_label`, `django_content_type`.`model` FROM `django_content_type` WHERE (`django_content_type`.`model` = 'service' AND `django_content_type`.`app_label` = 'portal'); args=('service', 'portal')
DEBUG 2016-03-31 07:03:06,015 utils (0.000) SELECT `django_content_type`.`id`, `django_content_type`.`app_label`, `django_content_type`.`model` FROM `django_content_type` WHERE (`django_content_type`.`model` = 'user' AND `django_content_type`.`app_label` = 'portal'); args=('user', 'portal')
DEBUG 2016-03-31 07:03:06,017 utils (0.001) SELECT `auth_permission`.`content_type_id`, `auth_permission`.`codename` FROM `auth_permission` INNER JOIN `django_content_type` ON (`auth_permission`.`content_type_id` = `django_content_type`.`id`) WHERE `auth_permission`.`content_type_id` IN (8, 7) ORDER BY `django_content_type`.`app_label` ASC, `django_content_type`.`model` ASC, `auth_permission`.`codename` ASC; args=(8, 7)
DEBUG 2016-03-31 07:03:06,019 utils (0.000) SELECT `django_content_type`.`id`, `django_content_type`.`app_label`, `django_content_type`.`model` FROM `django_content_type` WHERE `django_content_type`.`app_label` = 'portal'; args=('portal',)
DEBUG 2016-03-31 07:14:07,607 utils (0.000) SET SQL_AUTO_IS_NULL = 0; args=None
DEBUG 2016-03-31 07:14:07,657 utils (0.000) SET SQL_AUTO_IS_NULL = 0; args=None
DEBUG 2016-03-31 07:14:07,659 utils (0.001) SHOW FULL TABLES; args=None
DEBUG 2016-03-31 07:14:07,663 utils (0.001) SELECT `django_migrations`.`app`, `django_migrations`.`name` FROM `django_migrations`; args=()
DEBUG 2016-03-31 07:26:21,817 utils (0.000) SET SQL_AUTO_IS_NULL = 0; args=None
DEBUG 2016-03-31 07:26:21,866 utils (0.000) SET SQL_AUTO_IS_NULL = 0; args=None
DEBUG 2016-03-31 07:26:21,867 utils (0.001) SHOW FULL TABLES; args=None
DEBUG 2016-03-31 07:26:21,870 utils (0.000) SELECT `django_migrations`.`app`, `django_migrations`.`name` FROM `django_migrations`; args=()
DEBUG 2016-03-31 07:27:39,580 utils (0.000) SET SQL_AUTO_IS_NULL = 0; args=None
DEBUG 2016-03-31 07:27:39,634 utils (0.000) SET SQL_AUTO_IS_NULL = 0; args=None
DEBUG 2016-03-31 07:27:39,636 utils (0.001) SHOW FULL TABLES; args=None
DEBUG 2016-03-31 07:27:39,640 utils (0.001) SELECT `django_migrations`.`app`, `django_migrations`.`name` FROM `django_migrations`; args=()
WARNING 2016-03-31 07:27:42,658 base Not Found: /portal/submit
DEBUG 2016-03-31 07:27:42,662 base Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "/home/benz/venv/lib/python2.7/site-packages/django/template/base.py", line 905, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [name] in u'<RegexURLResolver <RegexURLPattern list> (admin:admin) ^admin/>'
ERROR 2016-03-31 07:27:42,664 base Internal Server Error: /portal/submit
Traceback (most recent call last):
  File "/home/benz/venv/lib/python2.7/site-packages/django/core/handlers/base.py", line 235, in get_response
    response = middleware_method(request, response)
  File "/home/benz/venv/lib/python2.7/site-packages/django/middleware/common.py", line 114, in process_response
    if self.should_redirect_with_slash(request):
  File "/home/benz/venv/lib/python2.7/site-packages/django/middleware/common.py", line 79, in should_redirect_with_slash
    and urlresolvers.is_valid_path('%s/' % request.path_info, urlconf)
  File "/home/benz/venv/lib/python2.7/site-packages/django/core/urlresolvers.py", line 668, in is_valid_path
    resolve(path, urlconf)
  File "/home/benz/venv/lib/python2.7/site-packages/django/core/urlresolvers.py", line 534, in resolve
    return get_resolver(urlconf).resolve(path)
  File "/home/benz/venv/lib/python2.7/site-packages/django/core/urlresolvers.py", line 376, in resolve
    sub_match = pattern.resolve(new_path)
  File "/home/benz/venv/lib/python2.7/site-packages/django/core/urlresolvers.py", line 248, in resolve
    return ResolverMatch(self.callback, args, kwargs, self.name)
  File "/home/benz/venv/lib/python2.7/site-packages/django/core/urlresolvers.py", line 255, in callback
    self._callback = get_callable(self._callback_str)
  File "/home/benz/venv/lib/python2.7/site-packages/django/utils/lru_cache.py", line 100, in wrapper
    result = user_function(*args, **kwds)
  File "/home/benz/venv/lib/python2.7/site-packages/django/core/urlresolvers.py", line 136, in get_callable
    (lookup_view, mod_name))
ViewDoesNotExist: Could not import 'portal.view.submit'. View does not exist in module portal.view.
DEBUG 2016-03-31 07:28:38,137 utils (0.000) SET SQL_AUTO_IS_NULL = 0; args=None
DEBUG 2016-03-31 07:28:38,185 utils (0.000) SET SQL_AUTO_IS_NULL = 0; args=None
DEBUG 2016-03-31 07:28:38,186 utils (0.001) SHOW FULL TABLES; args=None
DEBUG 2016-03-31 07:28:38,189 utils (0.000) SELECT `django_migrations`.`app`, `django_migrations`.`name` FROM `django_migrations`; args=()
WARNING 2016-03-31 07:28:40,712 base Not Found: /portal/submit
DEBUG 2016-03-31 07:28:40,716 base Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "/home/benz/venv/lib/python2.7/site-packages/django/template/base.py", line 905, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [name] in u'<RegexURLResolver <RegexURLPattern list> (admin:admin) ^admin/>'
ERROR 2016-03-31 07:28:40,722 base Internal Server Error: /portal/submit
Traceback (most recent call last):
  File "/home/benz/venv/lib/python2.7/site-packages/django/core/handlers/base.py", line 235, in get_response
    response = middleware_method(request, response)
  File "/home/benz/venv/lib/python2.7/site-packages/django/middleware/common.py", line 114, in process_response
    if self.should_redirect_with_slash(request):
  File "/home/benz/venv/lib/python2.7/site-packages/django/middleware/common.py", line 79, in should_redirect_with_slash
    and urlresolvers.is_valid_path('%s/' % request.path_info, urlconf)
  File "/home/benz/venv/lib/python2.7/site-packages/django/core/urlresolvers.py", line 668, in is_valid_path
    resolve(path, urlconf)
  File "/home/benz/venv/lib/python2.7/site-packages/django/core/urlresolvers.py", line 534, in resolve
    return get_resolver(urlconf).resolve(path)
  File "/home/benz/venv/lib/python2.7/site-packages/django/core/urlresolvers.py", line 376, in resolve
    sub_match = pattern.resolve(new_path)
  File "/home/benz/venv/lib/python2.7/site-packages/django/core/urlresolvers.py", line 248, in resolve
    return ResolverMatch(self.callback, args, kwargs, self.name)
  File "/home/benz/venv/lib/python2.7/site-packages/django/core/urlresolvers.py", line 255, in callback
    self._callback = get_callable(self._callback_str)
  File "/home/benz/venv/lib/python2.7/site-packages/django/utils/lru_cache.py", line 100, in wrapper
    result = user_function(*args, **kwds)
  File "/home/benz/venv/lib/python2.7/site-packages/django/core/urlresolvers.py", line 115, in get_callable
    mod = import_module(mod_name)
  File "/usr/lib64/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/home/benz/portal/views.py", line 11, in <module>
    from plugin import *
  File "/home/benz/portal/plugin.py", line 6, in <module>
    import requests
ImportError: No module named requests
DEBUG 2016-03-31 07:29:08,527 utils (0.000) SET SQL_AUTO_IS_NULL = 0; args=None
DEBUG 2016-03-31 07:29:08,577 utils (0.000) SET SQL_AUTO_IS_NULL = 0; args=None
DEBUG 2016-03-31 07:29:08,578 utils (0.001) SHOW FULL TABLES; args=None
DEBUG 2016-03-31 07:29:08,581 utils (0.000) SELECT `django_migrations`.`app`, `django_migrations`.`name` FROM `django_migrations`; args=()
WARNING 2016-03-31 07:29:10,044 base Not Found: /portal/submit
DEBUG 2016-03-31 07:29:10,049 base Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "/home/benz/venv/lib/python2.7/site-packages/django/template/base.py", line 905, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [name] in u'<RegexURLResolver <RegexURLPattern list> (admin:admin) ^admin/>'
ERROR 2016-03-31 07:29:10,094 base Internal Server Error: /portal/submit
Traceback (most recent call last):
  File "/home/benz/venv/lib/python2.7/site-packages/django/core/handlers/base.py", line 235, in get_response
    response = middleware_method(request, response)
  File "/home/benz/venv/lib/python2.7/site-packages/django/middleware/common.py", line 115, in process_response
    return self.response_redirect_class(self.get_full_path_with_slash(request))
  File "/home/benz/venv/lib/python2.7/site-packages/django/middleware/common.py", line 99, in get_full_path_with_slash
    'url': request.get_host() + new_path,
RuntimeError: You called this URL via POST, but the URL doesn't end in a slash and you have APPEND_SLASH set. Django can't redirect to the slash URL while maintaining POST data. Change your form to point to 127.0.0.1:8000/portal/submit/ (note the trailing slash), or set APPEND_SLASH=False in your Django settings.
WARNING 2016-03-31 07:32:16,843 base Not Found: /portal/submit
DEBUG 2016-03-31 07:32:16,847 base Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "/home/benz/venv/lib/python2.7/site-packages/django/template/base.py", line 905, in _resolve_lookup
    (bit, current))  # missing attribute
VariableDoesNotExist: Failed lookup for key [name] in u'<RegexURLResolver <RegexURLPattern list> (admin:admin) ^admin/>'
ERROR 2016-03-31 07:32:16,848 base Internal Server Error: /portal/submit
Traceback (most recent call last):
  File "/home/benz/venv/lib/python2.7/site-packages/django/core/handlers/base.py", line 235, in get_response
    response = middleware_method(request, response)
  File "/home/benz/venv/lib/python2.7/site-packages/django/middleware/common.py", line 115, in process_response
    return self.response_redirect_class(self.get_full_path_with_slash(request))
  File "/home/benz/venv/lib/python2.7/site-packages/django/middleware/common.py", line 99, in get_full_path_with_slash
    'url': request.get_host() + new_path,
RuntimeError: You called this URL via POST, but the URL doesn't end in a slash and you have APPEND_SLASH set. Django can't redirect to the slash URL while maintaining POST data. Change your form to point to 127.0.0.1:8000/portal/submit/ (note the trailing slash), or set APPEND_SLASH=False in your Django settings.
DEBUG 2016-03-31 07:32:51,047 utils (0.000) SET SQL_AUTO_IS_NULL = 0; args=None
DEBUG 2016-03-31 07:32:51,096 utils (0.000) SET SQL_AUTO_IS_NULL = 0; args=None
DEBUG 2016-03-31 07:32:51,098 utils (0.001) SHOW FULL TABLES; args=None
DEBUG 2016-03-31 07:32:51,101 utils (0.000) SELECT `django_migrations`.`app`, `django_migrations`.`name` FROM `django_migrations`; args=()
WARNING 2016-03-31 07:33:16,222 csrf Forbidden (CSRF cookie not set.): /portal/submit/
DEBUG 2016-03-31 07:33:54,690 utils (0.000) SET SQL_AUTO_IS_NULL = 0; args=None
DEBUG 2016-03-31 07:33:54,736 utils (0.000) SET SQL_AUTO_IS_NULL = 0; args=None
DEBUG 2016-03-31 07:33:54,738 utils (0.001) SHOW FULL TABLES; args=None
DEBUG 2016-03-31 07:33:54,740 utils (0.000) SELECT `django_migrations`.`app`, `django_migrations`.`name` FROM `django_migrations`; args=()
DEBUG 2016-03-31 07:33:57,868 utils (0.000) SET SQL_AUTO_IS_NULL = 0; args=None
DEBUG 2016-03-31 07:33:57,870 utils (0.000) INSERT INTO `portal_user` (`name`, `mobile`, `email`, `sex`, `province`, `city`, `service_name`) VALUES ('sf', '1232221112', 'fdsf', '1', '2', NULL, '南宁恒信之星'); args=[u'sf', u'1232221112', u'fdsf', u'1', u'2', None, u'\u5357\u5b81\u6052\u4fe1\u4e4b\u661f']
ERROR 2016-03-31 07:33:57,870 base Internal Server Error: /portal/submit/
Traceback (most recent call last):
  File "/home/benz/venv/lib/python2.7/site-packages/django/core/handlers/base.py", line 149, in get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/benz/venv/lib/python2.7/site-packages/django/core/handlers/base.py", line 147, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/benz/portal/views.py", line 59, in submit
    u.save()
  File "/home/benz/venv/lib/python2.7/site-packages/django/db/models/base.py", line 700, in save
    force_update=force_update, update_fields=update_fields)
  File "/home/benz/venv/lib/python2.7/site-packages/django/db/models/base.py", line 728, in save_base
    updated = self._save_table(raw, cls, force_insert, force_update, using, update_fields)
  File "/home/benz/venv/lib/python2.7/site-packages/django/db/models/base.py", line 812, in _save_table
    result = self._do_insert(cls._base_manager, using, fields, update_pk, raw)
  File "/home/benz/venv/lib/python2.7/site-packages/django/db/models/base.py", line 851, in _do_insert
    using=using, raw=raw)
  File "/home/benz/venv/lib/python2.7/site-packages/django/db/models/manager.py", line 122, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/benz/venv/lib/python2.7/site-packages/django/db/models/query.py", line 1039, in _insert
    return query.get_compiler(using=using).execute_sql(return_id)
  File "/home/benz/venv/lib/python2.7/site-packages/django/db/models/sql/compiler.py", line 1060, in execute_sql
    cursor.execute(sql, params)
  File "/home/benz/venv/lib/python2.7/site-packages/django/db/backends/utils.py", line 79, in execute
    return super(CursorDebugWrapper, self).execute(sql, params)
  File "/home/benz/venv/lib/python2.7/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/home/benz/venv/lib/python2.7/site-packages/django/db/utils.py", line 95, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "/home/benz/venv/lib/python2.7/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/home/benz/venv/lib/python2.7/site-packages/django/db/backends/mysql/base.py", line 112, in execute
    return self.cursor.execute(query, args)
  File "/home/benz/venv/lib/python2.7/site-packages/MySQLdb/cursors.py", line 205, in execute
    self.errorhandler(self, exc, value)
  File "/home/benz/venv/lib/python2.7/site-packages/MySQLdb/connections.py", line 36, in defaulterrorhandler
    raise errorclass, errorvalue
OperationalError: (1366, "Incorrect string value: '\\xE5\\x8D\\x97\\xE5\\xAE\\x81...' for column 'service_name' at row 1")
DEBUG 2016-03-31 07:33:57,918 utils (0.000) SELECT `portal_user`.`id`, `portal_user`.`name`, `portal_user`.`mobile`, `portal_user`.`email`, `portal_user`.`sex`, `portal_user`.`province`, `portal_user`.`city`, `portal_user`.`service_name` FROM `portal_user` LIMIT 21; args=()
DEBUG 2016-03-31 07:34:45,515 utils (0.000) SET SQL_AUTO_IS_NULL = 0; args=None
DEBUG 2016-03-31 07:34:45,516 utils (0.000) INSERT INTO `portal_user` (`name`, `mobile`, `email`, `sex`, `province`, `city`, `service_name`) VALUES ('sf', '1232221112', 'fdsf', '1', '2', NULL, '南宁恒信之星'); args=[u'sf', u'1232221112', u'fdsf', u'1', u'2', None, u'\u5357\u5b81\u6052\u4fe1\u4e4b\u661f']
